generator/AWSPSGeneratorLib/HelpMaterials/Examples/DS/Add-DSIpRoute.xml

```diff
@@ -0,0 +1,10 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<!--Examples for the AWS PowerShell CmdLets-->
+<examples>
+  <example>
+    <code>$IP = New-Object Amazon.DirectoryService.Model.IpRoute
+$IP.CidrIp = "203.0.113.5/32"
+Add-DSIpRoute -DirectoryId d-123456ijkl -IpRoute $IP -UpdateSecurityGroupForDirectoryController $true</code>
```

Comment:
This can be simplified with `Add-DSIpRoute -IpRoute @{CidrIp ="203.0.113.5/32"}`

Reply:
Thank you for reviewing this PR, Matteo. I have simplified the commands as you suggested. 
`Add-DSIpRoute -DirectoryId d-123456ijkl -IpRoute @{CidrIp ="203.0.113.5/32"} -UpdateSecurityGroupForDirectoryController $true`